sonar-csharp-plugin/src/test/java/org/sonar/plugins/csharp/CSharpSonarWayProfileTest.java

```diff
@@ -180,33 +177,19 @@ public void sonar_security_InvocationTargetException() {
     CsRules.exceptionToThrow = new InvocationTargetException(new Exception());
     CsRules.returnRepository = true;
 
-    CSharpSonarWayProfile profileDef = new CSharpSonarWayProfile(SQ_67);
+    CSharpSonarWayProfile profileDef = new CSharpSonarWayProfile(SonarVersion.SQ_79_RUNTIME);
     profileDef.define(context);
 
     assertThat(logTester.logs(LoggerLevel.DEBUG)).hasSize(1);
   }
 
   @Test
-  public void hotspots_not_in_sonar_way_before_SQ_73() {
```

Comment:
given you're dropping this, you should also remove the `supportsSecurityHotspots` field from [AbstractSonarWayProfile ](https://github.com/SonarSource/sonar-dotnet/blob/82cfe531bc5ec5d5ca3e88478d7c9f9f6b47de77/sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractSonarWayProfile.java) because

```
    this.supportsSecurityHotspots = sonarRuntime != null && sonarRuntime.getApiVersion().isGreaterThanOrEqual(SQ_7_3);
```

Reply:
and also, this means `Sonar_way_profile_no_hotspot.json` is not needed anymore, so there's some clean-up to do in scripts/rspec/rspec2resx.ps1 and scripts/rspec/rspec.ps1

Reply:
and also sonaranalyzer-dotnet/rspec/cs/Sonar_way_profile_no_hotspot.json  should be removed

Reply:
Nice catch!

Reply:
I've managed to remove quite a bit of code with this.